.github/workflows/ci.yml

```
@@ -5,9 +5,13 @@ jobs:
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@v2
-
-      - name: bundle install
-        run: bundle install
+      - uses: ruby/setup-ruby@v1
+      - uses: actions/cache@v1
+        with:
+          path: vendor/bundle
+          key: bundle-${{ hashFiles('**/Gemfile.lock') }}
+          restore-keys: bundle
+      - run: bundle install --deployment
```

```suggestion
      - run: bundle install --deployment
      - run: bundle install --jobs 4 --retry 3 --deployment
```

Will make it just a bit more consistent and a little faster.